inc/3rd-party/hosting/presslabs.php

```diff
@@ -0,0 +1,139 @@
+<?php
+defined( 'ABSPATH' ) || die( 'Cheatin&#8217; uh?' );
+
+if ( defined( 'PL_INSTANCE_REF' ) && class_exists( '\Presslabs\Cache\CacheHandler' ) ) {
+
+	add_action( 'pl_pre_cache_refresh', 'rocket_clean_post', 0 );
+	add_action( 'after_rocket_clean_domain', 'rocket_pl_clear_cache' );
+	add_action( 'after_rocket_clean_post', 'rocket_pl_clean_post', 2 );
+	add_filter( 'rocket_display_varnish_options_tab', '__return_false' );
+	add_filter( 'do_rocket_generate_caching_files', '__return_false', PHP_INT_MAX );
+	add_filter( 'rocket_cache_mandatory_cookies', '__return_empty_array', PHP_INT_MAX );
+	add_action( 'after_rocket_clean_home', 'rocket_pl_clean_home', 10, 2 );
+	add_action( 'after_rocket_clean_file', 'rocket_pl_clean_url' );
+	add_action( 'pl_pre_url_button_cache_refresh', 'after_rocket_clean_file' );
```

Comment:
`after_rocket_clean_file` is not a callable. It should be `rocket_clean_files()`

Reply:
Ok, updated!